front/components/vaults/VaultResourcesList.tsx

```diff
@@ -112,7 +112,13 @@ const getTableColumns = ({
         : row.dataSourceView.editedByUser?.imageUrl) ?? "",
     id: "managedBy",
     cell: (info: CellContext<RowData, string>) => (
-      <DataTable.CellContent avatarUrl={info.getValue()} roundedAvatar={true} />
+      <DataTable.CellContent
+        avatarUrl={info.getValue()}
+        avatarTooltipLabel={
+          info.row.original.dataSourceView.editedByUser?.fullName ?? undefined
```

Comment:
we have a switch if it's the default view, we show the `dataSourceView.dataSource.editedByUser` image , not the `dataSourceView.editedByUser` ( as nobody can edit the defaut view, we show the datasource editor)

Reply:
Thanks a lot, don't know how I missed that since the switch is 3 lines before the code I edited 🤦🏻‍♀️